src/main/webapp/coffeeshop.html

```diff
@@ -1,26 +1,62 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Coffee Shop Ratings</title>
-  <script src="https://www.gstatic.com/charts/loader.js"></script>
-  <script src="review.js"></script>
-</head>
-<body>
-  <a href="index.html" class="button">Home</a>
-  <a href="user.html" class="button">User</a>
-  <a href="store.html" class="button">Stores</a>
-  
-  <h1>Coffee Shop Ratings</h1>
-  <div id="chart-container"></div>
+  <head>
+    <meta charset="UTF-8">
+    <title>Coffee Shop Ratings</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="review.js"></script>
+    <script src="script.js"></script>
+  </head>
+ 
+  <body onload="loadComments()">
+    <a href="store.html" class="button">Back</button>
+    <a href="index.html" class="button">Home</a>
+    <a href="user.html" class="button">User</a>
+ 
+    <h1>Coffee Shop Ratings</h1>
+    <div id="chart-container"></div>
+ 
+    <h2>Comments</h2>
+ 
+    <form action="/make-comment" method="POST">
+    <textarea name="text-input"></textarea>
+    <input type="submit"/>
+    </form><br>
+ 
+    <p>Rate this store:</p>
+    <form action="/store-rating" method="POST">
+    <select name="store-rating">
+        <option value="5">5</option>
+        <option value="4">4</option>
+        <option value="3">3</option>
+        <option value="2">2</option>
+        <option value="1">1</option>
+    </select>
+    <input type="submit"/><br>
+    </form>
+    <br/><br/>
 
-  <h2>Comments</h2>
-  <form action="/make-comment" method="POST">
-  <textarea name="text-input"></textarea>
-  <br>
-  <input type="submit"/>
-  </form>
-  <br>
-    <button onclick="loadComments()">Click here to see comments</button>
+    <!-- TODO: RATE DRINKS,SHOW THEIR RATINGS -->
+    <form action="/drink-rating" method="POST">
+      <p>Enter your drink below:</p>
+      <textarea name="drink"></textarea>
+
+      <p>Rate your drink:</p>
+      <select name="drink-rating">
+        <option value="5">5</option>
+        <option value="4">4</option>
+        <option value="3">3</option>
+        <option value="2">2</option>
+        <option value="1">1</option>
+      </select>
+      <input type="submit" />
+    </form>
+    <button onclick="loadRatings()">Click here to see ratings</button>
```

Comment:
Should we have a button to see ratings or would it be easier to automatically display them through an onload function?

Reply:
We plan on having it automatically displayed.